.github/workflows/backend-test.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will run backend tests on the Python version defined in the Dockerfiles
+
+name: Backend unit tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'develop'
+      - 'master'
+      - 'feature/**'
+      - 'bugfix/**'
+      - 'hotfix/**'
+      - 'release/**'
+      - 'dependabot/**'
+    paths:
+      - 'backend/**'
```

Comment:
This would also ignore changes to `docker-compose.yaml` or the workflow configuration, which could affect the test outcome.

Reply:
I was actually wondering whether the global `package.json` shouldn't also be watched, as the commands in that may change.

Reply:
This is why I had suggested `ignore-paths` originally. I think excluding paths is less likely to cause false negatives, especially as the repository is updated over time.

Anyway, a more serious issue is that skipping these checks does not seem to play well with branch protections.

Reply:
I'll look into it. I thought `ignore-paths` was going to get wordier than `paths`, but I'll see how it plays out.